back-end/middlewares/auth.js

```diff
@@ -4,8 +4,8 @@ module.exports = async (req, res, next) => {
     res.send("login first");
   }
   const token = req.headers.authorization.split(" ").pop();
-  jwt.verify(token, process.env.SECRET, (err, parsedToken) => {
-    if (err) res.send(err);
+  jwt.verify(JSON.parse(token), process.env.SECRET, (err, parsedToken) => {
+    if (err) res.send(err.message);
     if (parsedToken) {
       next();
```

Comment:
Before invoking the next function you can do what I mentioned above about the token